tests/test_examples.py

```
@@ -0,0 +1,455 @@
+    """The examples submodule is part of the package's public surface."""
+
+    def test_examples_is_bound_on_the_package(self):
+        import sans_fitter
+
+        assert sans_fitter.examples is examples
+
+    def test_star_import_provides_every_all_entry(self):
+        import sans_fitter
+
+        namespace: dict = {}
+        exec('from sans_fitter import *', namespace)
```

Ruff (0.16.1)

[error] Use of `exec` detected

(S102)